.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: node_js
+node_js:
+  - "8.9.4"   # Likely set Azure webapp minimum
+  - "8.11.1"  # Highest 8.* in Azure
+  - "lts/*"   # Latest Node 8.* LTS
```

Comment:
this doesn't ensure its 8, that could move at any time?

Reply:
I didn't try an `lts/8` tbh.  Kinda because I assumed that when that actually does happen we'd want to be on that LTS anyway so I wasn't too concerned.

Reply:
ok then could you update the comment to remove 8.x,